sites/spiano.dev.md

```diff
@@ -0,0 +1,7 @@
+---
+title: 'spiano.dev'
+url: 'https://www.spiano.dev/'
+tags: ['web developer', 'full stack', 'software engineer]
```

Comment:
Just add the closing `'`, so it’s `'software engineer']`

Reply:
All mistakes have been corrected in today's commit. Sorry about that!

Reply:
Fabulous! No need to apologise—I normally don't bother checking and keep tanking the site 😂